### PR TITLE
Add documentation for the scripts used to analyze/annotate the logs

### PR DESCRIPTION
**Pull request by gcampax**
For myself and for others in the team who will be looking at logs.

@kevintangzero This PR adds some documentation for what I currently have as workflow
to analyze the conversation logs, both the detailed ones that we get with "Enable recording"
and the passive command logs that we get from almost all users.

The `analyze-recordings.sh` script is built for file-base logs and will need to be replaced
with the script you're building, but the rest might still be helpful.

I imagine you will also want to build some tools to automate some of this analysis, but
I think it's good to have these scripts for manual analysis under your belt.

### CODE REVIEW

File: doc/log-processing.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Processing and Analyzing Conversation Logs
+
+This document details the procedures to analyze the commands
+collected from users who have agreed to participate in our
+research and share their commands with us.
+
+There are two type of logs:
+- "passive" logs, which contain only the delexicalized context
+  + command. These logs are enabled by agreeing to the consent form
```
Comment (kevintangzero) [inline doc/log-processing.md:9]: Is "+" a typo?

File: doc/log-processing.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+or comments. After looking at the logs, the files should be moved to `./logs/seen/`.
+
+### Step 2: Annotate the logs
+
+```bash
+./scripts/recording-to-devset.js
+```
+
+The interface is similar to `genie manual-annotate-dialog` and `./scripts/annotate-log.js`.
+Each dialogue in the log is annotated until there is a parsing error
```
Comment (kevintangzero) [inline doc/log-processing.md:90]: I don't understand why there will be a parsing error.

**Review by kevintangzero — APPROVED** [review]
LGTM

File: doc/log-processing.md (hunk 2 above)
Comment (gcampax) [reply doc/log-processing.md:90]: I meant a semantic parsing error, that is, until the model interpreted the sentence wrong.

File: doc/log-processing.md (hunk 1 above)
Comment (gcampax) [reply doc/log-processing.md:9]: No it's "context + command", just wrapped weird. Let me fix that...

**Pull request merged**